packages/fastify/src/index.ts

```diff
@@ -27,7 +27,7 @@ export interface FastifyHandlerOptions<TContext extends BaseContext> {
 
 export function fastifyHandler(
   server: ApolloServer<BaseContext>,
-  options?: FastifyHandlerOptions<BaseContext>,
+  options?: never,
```

Comment:
What's going on here? Why not just drop the argument?

Reply:
This snuck in, a stray change from when I was playing around trying to improve the types